### ISSUE TITLE
Ace Finding

### ISSUE THREAD

**Issue by tfgrime** [HereBeMike/Gridcannon#6]
Awesome work Mike.  Since you were thinking about building something more robust and putting it on Steam I thought I might give you some suggestions for additional features to think about.

I've won my last 10 games in a row with 0 shame pile with a strategy that focuses heavily on minimizing the total value of the board and playing Aces on Aces as often as possible.
- Is there a way to display the cards in a given pile?
- If not, is there a way to make an indicator for if a pile contains an Ace?

I need something to make the game a little harder.  Things I've thought about:
- Option to remove Jokers from the deck
- Option to make aces not return to the deck when playing an ace on them

**Comment by HereBeMike** [comment]
Yeah, I've found the game is generally too easy as well. In the new build I'm working on, I've removed jokers all together.
The grid being 9x9 and only having 4 aces to keep track of, I decided not to indicate where aces are. It adds a little working memory challenge which I think is probably better than actively telling the player where to optimally place an ace.

I also made using the shame pile more restrictive - it has 16 shame points available. So, if the player discards cards exceeding 16 in total face value, the game is lost. I also changed the deck exhaustion rule, so that results in a loss as well.

I have given thought to the ace-cycling problem as well, and I'm not sold on penalising it. The strategy allows for a shameless run, if used effectively. However, I have added a mode where the aces start out of play and can be unlocked when the player kills two corner-adjacent royals. This adds a bit of early challenge that limits access to ace cycling in the early game, which usually leads to taking on more shame early on.

Once I finish up with the new build I'll probably add a mode-switch to the javascript version so that the newer rules I've worked on will be reflected here. Like an "orignal" and "enhanced" mode, or something.

Oh, and yes - to answer your question - we could add an indication of what cards are in which piles. To me, it would seem to make things a little too easy, but maybe the burden of working memory isn't fun... So I'll consider that in future as well.

Thanks for the feedback and suggestions!
